architecture/terraform.tfstate

```diff
@@ -0,0 +1,851 @@
+{
```

Comment:
do not upload your tf state file. It could include sesnstive informations about your system

Reply:
better use s3 as backend config for your terraform state